chipy_org/apps/meetings/models.py

```diff
@@ -236,9 +271,25 @@ def __str__(self):
 
 class RSVP(CommonModel):
 
-    RSVP_CHOICES = (
-        ("Y", "Yes"),
-        ("N", "No"),
+    YES = "Y"
+    NO = "N"
+    RSVP_RESPONSE = ((YES, "Yes"), (NO, "No"))
```

Comment:
Would recommend naming these as plural of the name of the name of the field, i.e. `RESPONSES` or whatever you wind-up with. 

I also recommend doing nested classes for this: 

```
class RSVP(CommonModel):
  class Response:
    YES = "yes"
    NO = "no"
    ALL = [YES, NO]
    CHOICE_LIST = [(YES, "YES"), (NO, "NO")]
```

Reply:
Good call.